source/usage-examples/findOneAndDelete.txt

```diff
@@ -0,0 +1,81 @@
+===================
+Delete One Document
+===================
+
+.. default-domain:: mongodb
+
+Overview
+--------
+
+You can delete a single document in a collection with
+``collection.findOneAndDelete()``.
```

Comment:
This seems to bury the lede on the difference between `collection.findOneAndDelete()` and [collection.deleteOne()](http://mongodb.github.io/node-mongodb-native/3.2/api/Collection.html#deleteOne). Is there a way to clarify why you would specifically use `findOneAndDelete()` here?